tests/acceptance/poc-test.js

```diff
@@ -1,23 +0,0 @@
-import Ember from 'ember';
```

Comment:
cool, I didn't think of that. Since mocha wants `/test` and ember wants `/tests` (by default) they shouldn't conflict with each other)